### ISSUE TITLE
Des suggestions

### ISSUE THREAD

**Issue by couleurm** [Piwielle/Backup-logiciels#1]
Salut, je te laisse ces deux merveilles pour unifier et synchroniser tes appareils:

[Scoop](https://github.com/ScoopInstaller/Scoop), le meilleur package manager pour moi, il garde tout en un seul fichier, et tu peux faire [tes propres installeurs](https://github.com/couleur-tweak-tips/utils/tree/main/bucket) (app manifests) 
Syncthing
[Syncthing](https://github.com/syncthing/syncthing) peut synchroniser tes apparails ensemble, tu peux choisir par dossier/mettre des blacklist

Tu peux facilement mettre les deux en symbiose (syncthing s'installe avec scoop si tu as le bucket `extras`) pour avoir les mêmes logiciels partout

**Comment by Piwielle** [comment]
ça c'est chouette merci
j'ai déjà un peu expérimenté avec chocolatey/scoop/winget, ça a toujours été "pas mal, mais pas encore vraiment au points sur windows"
je réessaierai scoop en le personnalisant à l'occasion

**Comment by couleurm** [comment]
je suis d'accord avec toi pour scoop, faut faire son prope bucket pour les trucs qui manquent c'est amusant
